CryptoWallet/Data/Model/AssetMarket.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Market.swift
+//  CryptoWallet
+//
+//  Created by Phan Dinh Van on 4/1/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+final class AssetMarket: BaseModel {
+    var websiteSlug: String?
+    var lastUpdated: Double?
```

Comment:
```suggestion
    var lastUpdated: Double = 0.0
```
tương tự với các cái bên dưới